docs/token-metadata/specification.md

```diff
@@ -342,3 +342,35 @@ This allows projects to set different limits on usage of gaming tokens: burn, si
 ### Delegate Use Authority
 
 Owners of NFTs can now allow a program to `Use` their token without them being online. This is available via the `approve_use_authority` instruction. It is very similar to the Collection Authority system but the party who can approve and revoke is the current holder of the NFT.
+
+
+### Verify a collection using collections.metaplex.com
+
+1. Visit collections.metaplex.com[https://collections.metaplex.com/]
```

Comment:
Correct markdown format for a link is like this:

`Visit [collections.metaplex.com](https://collections.metaplex.com/)`

Reply:
done thank you!